ruby/pom.xml

```diff
@@ -86,7 +86,7 @@
         <dependency>
             <groupId>com.google.protobuf</groupId>
             <artifactId>protobuf-java</artifactId>
-            <version>3.0.0-beta-3</version>
+            <version>3.0.0</version>
```

Comment:
This will need to stay as 3.0.0-beta-4 until we release Java 3.0.0 to maven. Changing it to 3.0.0 now will break the jruby build.